src/include/stir/ProjData.h

```diff
@@ -151,6 +152,10 @@ class ProjData : public ExamData
   //! Set sinogram
   virtual Succeeded 
     set_sinogram(const Sinogram<float>&) = 0;
+  //! Get Bin value
+   virtual float get_bin_value(const int view_pos, const int segment_pos, const int axial_pos, const int tang_pos) const = 0;
+  //! Get Bin value
+  virtual float get_bin_value(const Bin& this_bin) const = 0;
```

Comment:
Unless we really need this I would comment this out for the moment. It breaks compatibility for all those non-ProjDataFromStream objects.
we could flag it up as something to implement in the future.